persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/Visit.java

```diff
@@ -1,19 +1,13 @@
 package cz.muni.fi.pa165.mushrooms.entity;
 
+import com.sun.javafx.UnmodifiableArrayList;
 import cz.muni.fi.pa165.mushrooms.utils.LocalDateAttributeConverter;
 
-import javax.persistence.Column;
-import javax.persistence.Convert;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.Table;
-import javax.persistence.UniqueConstraint;
+import javax.persistence.*;

```

Comment:
we might want to avoid * imports, but w/e

Reply:
resolved